### DISCUSSION TITLE
Can we have event viewer integration?

### DISCUSSION THREAD

**Discussion by KrisTiteca** [microsoft/winget-cli#3119] [Ideas]
I've looked under ideas but couldn't find any similar idea.

From a secops and enterprise management point of view it would be much more valuable that any winget actions are logged into the event viewer. Ideally in it's own dedicated eventlog, if not possible I would say the standard application log but with its own event source. 

This would enable a lot of scenario's from forwarding events with tools like splunk or even Defender for endpoint to be able to collect those events. But it also would allow better troubleshooting and being able to track the history and it would even allow to attach tasks to certain events.

Thanks,
Kris Titeca

**Comment by Trenly** [comment]
I've referenced this discussion as a new [Feature Request](https://github.com/microsoft/winget-cli/issues/3125)
